libdnf/dnf-version.h.in

```diff
@@ -37,6 +37,7 @@
 #define LIBDNF_MAJOR_VERSION				(@LIBDNF_MAJOR_VERSION@)
 #define LIBDNF_MINOR_VERSION				(@LIBDNF_MINOR_VERSION@)
 #define LIBDNF_MICRO_VERSION				(@LIBDNF_MICRO_VERSION@)
+#define LIBDNF_VERSION					"@LIBDNF_VERSION@"
```

Comment:
Why does this need to exist? Can't you just construct this from the substitutions above?

Reply:
I initially did just that, but then realized that we already define the LIBDNF_VERSION cmake variable. Plus, there are more instances in the code where we construct the full version string, so it feels redundant to always have to do the concatenation. But yeah, I'm indifferent here :) I might as well construct it if that saves us one extra constant :)